dashboard/app/models/karel.rb

```diff
@@ -22,10 +22,10 @@
 #
 
 # Used by Bee to indicate a flower/honeycomb of capacity 0
-ZERO_DIRT_ITEM = 98
+#ZERO_DIRT_ITEM = 98
```

Comment:
oh, yes; good catch! Since we're not doing parsing here in rails anymore, it no longer needs specialized knowledge.